tests/improver-weighted-blending/02-basic_nonlin.bats

```diff
@@ -36,7 +36,7 @@
   KGO="weighted_blending/basic_nonlin/kgo.nc"
 
   # Run weighted blending with non linear weights and check it passes.
-  run improver weighted-blending 'nonlinear' 'forecast_reference_time' 'weighted_mean'\
+  run improver weighted-blending --wts_calc_method 'nonlinear' 'forecast_reference_time' 'weighted_mean'\
       "$IMPROVER_ACC_TEST_DIR/weighted_blending/basic_lin/multiple_probabilities_rain_*H.nc" \
```

Comment:
These input files contain out-dated meta-data. It would be good to update this.

Reply:
As discussed: I'd like to do this as a separate bit of work once the feature branch is merged into master, so that we're not changing the reference tests at the same time as changing functionality.  I'll open a ticket for this.